src/hooks/useHttpClient.tsx

```diff
@@ -48,7 +48,10 @@ export const HttpClientContextProvider = ({
   const { authResult, refreshForAuthFailure } = useAuth();
 
   const axiosInstance = injectedAxiosInstance || defaultAxiosInstance;
-  const apiClient = defaultAPIClientInstance;
+  const apiClient = useMemo(
+    () => new APIClient<RestAPIEndpoints>(axiosInstance),
+    [axiosInstance],
+  );
```

Comment:
Why is this needed?

Reply:
I wanted to provide an app config and it uses the new one query rest client that didn't work with the DataProviderDecorator prior to this change. I could probably back this out and just use the AppConfigContext.Provider instead.

Reply:
My gut reaction would be yes, if this is only beneficial for storyboard purposes, it'd be sweet to refactor to where this change isn't needed.  At the same time, I'm not overly concerned about the change, so if it's a can of worms nbd.

Reply:
Not a big deal. I switched it to just providing my own config